app/src/actions/user.js

```diff
@@ -51,6 +51,7 @@ export function logout() {
     dispatch({
       type: LOGOUT
     });
+    window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
```

Comment:
I could use the regex in App.jsx, but it looks like it's broken, and I don't dare touching it